ROMFS/px4fmu_common/init.d/airframes/13015_jiyi_batman

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# @name JIYIRobot Batman
+#
+# @type VTOL Tiltrotor
+# @class VTOL
+#
+# @maintainer Stone White <stone@thone.io>
+# @creator Yu Liu <lydoraemon@gmail.com>
```

Comment:
The `@creator` tag doesn't exist and it's resulting on CI failing on the Metadata job.

Reply:
Fixed

Reply:
Due to I'm maintaining and selling this airframe, I'll only keep my name.